### ISSUE TITLE
Add (etc) album to queue

### ISSUE THREAD

**Issue by quite** [jmshrv/finamp#221]
Just like we can add a song to the queue (or replace queue/add to playlist), it would be great to be able to do operations on album in a similar manner.

Long-pressing album in album list (and in list of albums by artist) should probably show these operations. And as well somewhere in the album screen. Long-press on header at the top? Or on the Play-button?

The operations should probably be Add To Queue, Replace Queue (well that would be the same as "Play" in album context, I guess), Add To Playlist, and maybe also Goto Artist.

**Comment by UnicornsOnLSD** [comment]
That would be a pretty useful feature, I'll add it :)

**Comment by BobcatNoah** [comment]
I'm still new to github, but shouldn't this issue be closed because it's been implemented?

**Comment by Chaphasilor** [comment]
Once #484 is merged, this will be implemented. The options shown on the album screen or when long-pressing an album in a list will be (in addition to the current ones): "Play next", "Add to Next Up", "Shuffle next", "Shuffle to Next Up". Adding a simple "Play" and "Shuffle" option to the long-press menu would also be possible, but this wasn't available before and therefore hasn't been changed.  
Adding an album to a playlist is not handled yet, and there is a separate issue for this (#497). 

Also @BobcatNoah you are right, although the previous PR targeted the legacy design and this new PR is a more extensive change targeting the same issue, that why this issue hasn't been closed yet. It will be closed once #484 has been merged :)

**Comment by kuflierl** [comment]
Alright, I think that issue has been mostly solved. I am still looking forward to a feature to add the queue to a playlist/make the queue a playlist

**Comment by Chaphasilor** [comment]
@kuflierl that's a different feature, but you could create a separate issue for it!  

I'll close this issue, since all features except "add to playlist" are implemented in the redesign, and for that there's a separate issue: #497

**Comment by kuflierl** [comment]
Alright
